src/app/shared/date/index.ts

```diff
@@ -0,0 +1 @@
+export * from './timeago.pipe';
```

Comment:
Hrmm... yeah, I don't have a good reason.  I've written that `time-ago-in-words` code too many times in too many languages.  Will pull in moment!